apps/voting-disputable/app/src/components/DisputableStatusLabel.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import {
+  GU,
+  IconAttention,
+  IconClock,
+  IconClosed,
+  IconInfo,
+  Tag,
+  textStyle,
+  useTheme,
+} from '@aragon/ui'
+import {
+  VOTE_STATUS_ACTIVE,
+  VOTE_STATUS_CANCELLED,
+  VOTE_STATUS_CLOSED,
+  VOTE_STATUS_PAUSED,
+  DISPUTABLE_VOTE_STATUSES,
+} from '../disputable-vote-statuses'
+
+function getAttributes(status, theme) {
+  const attributes = {
+    [VOTE_STATUS_ACTIVE]: {
+      label: 'Scheduled',
+      Icon: IconClock,
+    },
+    [VOTE_STATUS_CANCELLED]: {
+      background: String(theme.surfaceUnder),
```

Comment:
We can remove these string conversions as they are already being performed in the template string when passed to the component.

Reply:
I passed down like this cause theme.surfaceUnder is an extended String object, and on this Tag component is seen as an object instead of a normal string and throughs an error

Reply:
Interesting! the template string that's being passed to the component at the moment should already be converting this so you shouldn't be seeing any errors by removing them 🤔 .

I did find an error in this file with an icon import though which might be causing the problems! We're importing `IconClosed` which should probably be `IconClose`

Reply:
Yes I modified it, thanks :)